bcrhp/src/bcrhp/schema/SiteClassificationSchema.ts

```diff
@@ -7,7 +7,7 @@ const SiteClassificationSchema = z.object({
 });
 const HeritageClassSchema = z.object({
     contributingResources: z
-        .number()
+        .string()
```

Comment:
I think this should remain as a number. If we need to do a string->number conversion that should happen in the component or the step.